docs/guide/index.md

```diff
@@ -0,0 +1,79 @@
+# ds-cookiecutter guide (Kuebiko)
+
+> Kuebiko (久延毘古) is the Shinto kami ("god; deity") of Folk wisdom, knowledge and agriculture, and is represented in Japanese mythology as a scarecrow who cannot walk but has comprehensive awareness.
+
+A "full" (see [Corners cut](#corners-cut)) example [`ds-cookiecutter`](http://nestauk.github.io/ds-cookiecutter/) project broken down into episodes, with annotations/narrative for each episode focusing on a different aspect. In addition, there are a set of reference guides consolidating key concepts and pointing to further resources.
+
+!!! warning "Not from around these parts? A disclaimer..."
+
+    This is an **internal (but open) project** to develop and refine best practices for _a subset_ of Nesta's work delivering data science projects.
+
+    It is not a guide to production data science and the **recommended practices in this guide may be wholly unsuitable for you/your company** because you may be delivering completely different types of projects under completely different constraints.
+
+    Furthermore, these docs are deployed when a new episode is ready for review therefore **content is likely to change significantly**! To get an indication of what content is stable, check out which [episodes have closed issues](https://github.com/nestauk/kuebiko/issues?q=is%3Aclosed+label%3Aepisode+sort%3Acreated-asc+).
+
+### Project problem statement
+
+You have been given a big list (>100,000) of URLs corresponding to the business websites of UK companies.
+The stakeholder is interested in better understanding the performance, accessibility, and legal compliance of business websites across different industries (SIC codes).
+
+Some specific questions they have are:
+
+1. It is a [requirement](https://www.gov.uk/running-a-limited-company/signs-stationery-and-promotional-material) that limited companies must display the company's registered number on their website, how many businesses actually list a company number?
+2. Does the download size, response time, or reachability of a business' website bear any relation to the number of web development businesses nearby?
+3. How do web development businesses themselves do on the extracted features mentioned above?
+4. Is there any relationship between aforementioned extracted features and text on a business' website relating to what a business does?
+
+### Episode plan
+
+Below is a list of provisional episodes. See [issues labeled `episode`](https://github.com/nestauk/kuebiko/issues?q=is%3Aopen+label%3Aepisode+sort%3Acreated-asc) for more information and more timely updates.
+
+-   [x] 0. Setup
```

Comment:
I was a bit confused here, as I couldn't find "00 - Setup" on the left hand side of the website (Provided that ”0. Setup” is marked here as [x], I thought that the episode page should also exist)